src/cml.js

```diff
@@ -69,8 +70,41 @@ const getDriver = (opts) => {
   throw new Error(`driver ${driver} unknown!`);
 };
 
+const fixGitSafeDirectory = () => {
+  const getOrSetGitConfigSafeDirectory = (value) =>
+    spawnSync(
+      'git',
+      [
+        'config',
+        '--global',
+        value ? '--add' : '--get-all',
+        'safe.directory',
+        value
+      ],
+      {
+        encoding: 'utf8'
+      }
+    ).stdout.split(/[\r\n]+/);
+
+  const addSafeDirectoryIdempotent = (directory) =>
+    getOrSetGitConfigSafeDirectory().includes(directory) ||
+    getOrSetGitConfigSafeDirectory(directory);
+
+  addSafeDirectoryIdempotent('/');
+  addSafeDirectoryIdempotent('*');
```

Comment:
it's supposed to cover `/` and everything else, no?

Reply:
* `*` fixes the issue completely on `git>2.36.0`
* `/` and the loop do the same for `git>2.35.2,<2.36.0`

Reply:
Clarified with https://github.com/iterative/cml/pull/974/commits/7fa9da6e5dd1fc4480c336109524bfeeb0b1bca1